workflows/tests/test_author_tasks.py

```diff
@@ -85,13 +85,29 @@ def test_set_submission_number_with_data(self):
             == self.context["params"]["workflow_id"]
         )
 
-    @pytest.mark.skip(
-        reason="Test does not verify any behavior or assertions correctly."
-    )
     @pytest.mark.vcr
     def test_create_author_create_user_ticket(self):
+        noticket_context = {
+            "params": {
+                "workflow_id": "00000000-0000-0000-0000-000000001521",
+                "workflow": {
+                    "data": {
+                        "$schema": "https://inspirehep.net/schemas/records/authors.json",
+                        "_collections": ["Authors"],
+                        "name": {"preferred_name": "Third B", "value": "B, Third"},
+                        "acquisition_source": {"email": "micha.moshe.moskovic@cern.ch"},
+                    },
+                    "decisions": [],
+                    "id": "66277811-fe66-4335-9aff-984583fb1228",
+                    "status": "running",
+                    "tickets": [],
+                    "workflow_type": "AUTHOR_CREATE",
+                },
+            }
+        }
+
         task = self.dag.get_task("create_author_create_user_ticket")
-        task.execute(context=self.context)
+        task.execute(context=noticket_context)
```

Comment:
I am not sure we need this test at all. Currently it tests that part here https://github.com/inspirehep/inspirehep/blob/b4dbe7ce7295d5ddbad17b628840dea49ef9a441/workflows/dags/author/author_create/author_create_init.py#L68-L72. WDYT ?